test/integration/bundle/test_download.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+import platform
+from semantic_version import Version
+import sys
+
+import pytest
+
+from release_creation.bundle.bundle_config import get_bundle_config
+from release_creation.bundle.create import _download_packages
+
+
+@pytest.mark.parametrize("test_version", ["0.0.0"])
```

Comment:
We can't currently test `1.8.0b1` since we know this will fail due to the metadata issue in the versions that's published to PyPI. We need to add it once we publish the next `dbt-postgres` beta.